cypress/e2e/shared/tasks.test.ts

```diff
@@ -211,26 +211,45 @@ from(bucket: "defbuck")
     })
 
     it('can clone a task and activate just the cloned one', () => {
-      createTask('task1', 'buckets()')
```

Comment:
There is already an existing task in the test suite. There is no need to create another task. We're testing labels and cloning here, not task creation.